.travis.yml

```diff
@@ -30,7 +30,7 @@ notifications:
     on_success: never
 before_install:
 #  - rustup target add $TARGET || true
-  - travis_wait cargo install rustfmt --force --vers 0.8.4 || true
+  - travis_wait cargo install rustfmt --force || true
```

Comment:
Mind to add the version? It would work better to add `--vers 0.9.0` so it won't break again in the future.